examples/tv-app/android/java/ContentLauncherManager.cpp

```diff
@@ -351,3 +155,124 @@ ContentLaunchResponse ContentLauncherManager::LaunchUrl(const chip::CharSpan & c
 
     return response;
 }
+
+std::list<std::string> ContentLauncherManager::HandleGetAcceptsHeaderList()
```

Comment:
Shouldn't this be `HandleGetAcceptHeaderList`?  The attribute is called "AcceptHeader"; the HTTP header is called "Accept".  There should be no "Accepts" around.

Reply:
Updated.